src/Makefile

```diff
@@ -2852,302 +2852,302 @@ auto/gui_gtk_gresources.h: gui_gtk_res.xml $(GUI_GTK_RES_INPUTS)
 # specified for each file separately.
 
 objects:
-	mkdir objects
+	mkdir -p objects
 
-objects/arabic.o: arabic.c
+objects/arabic.o: objects arabic.c
```

Comment:
You should use an [order-only dependency](https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html) so this file is not rebuilt when the directory timestamp changes:

```make
objects/arabic.o: arabic.c | objects
        ...
```

Although TBH I'm not sure if that's a GNU Make extension, or if it's portable enough to be acceptable to Vim...